stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageReactionItem.kt

```diff
@@ -19,10 +19,9 @@ public fun MessageReactionItem(
     option: ReactionOptionItemState,
     modifier: Modifier = Modifier,
 ) {
-    Icon(
+    Image(
```

Comment:
We could use an `Icon` still but like so:

```
    Icon(
        modifier = modifier,
        painter = option.painter,
        contentDescription = null,
        tint = Color.Unspecified
    )
```

This makes it so that the tint isn't applied and we can show small image icons (like emoji, custom images..)

Reply:
I inspected the source code. I think Image is even more lightweight that the Icon.